src/detect-bytetest.c

```diff
@@ -366,9 +368,9 @@ static DetectBytetestData *DetectBytetestParse(const char *optstr, char **value,
 
     if (test_value) {
         /*
-         * test_value was created while fetching strings and contains the test value and offset, comma separated. The
-         * values was allocated by test_value (pcre_get_substring) and data_offset (SCStrdup), respectively; e.g.,
-         * test_value,offset
+         * test_value was created while fetching strings and contains the test value and offset,
+         * comma separated. The values was allocated by test_value (pcre2_substring_get_bynumber)
```

Comment:
nit: value is

Reply:
I did not change that part of the comment
@jlucovsky your English is better than mine ;-) What is expected here ?